__tests__/Goals/components/TodoComponent-text.tsx

```diff
@@ -25,14 +25,19 @@ describe('TodoComponent', () => {
   });
 
   test.skip('calls navigationProp.navigate when "Add" button is pressed', async () => {
```

Comment:
why this test is skipped ?

Reply:
Hi @shreya-mishra If you look at it, this test was already there. I just have refactored it because it was causing warning.

Reply:
if we remove the skip then test will be failing right ?

Reply:
> hen test will be failing right ?

@shreya-mishra No, It is not.

Reply:
then can you please remove skip and then add the test coverage again ?

Reply:
@shreya-mishra There are more skip tests. Should I remove all instance?

Reply:
remove it atleast for this file and please check if tests are passing